packages/kbn-analytics/src/reporter.ts

```diff
@@ -63,14 +64,14 @@ export class Reporter {
     this.storageManager.store(this.reportManager.report);
   }
 
-  public start() {
+  public start = () => {
```

Comment:
I believe it's different in that it binds the context, but not sure about the lack of uniformity around this approach throughout Kibana

Reply:
if someone does this:

```
const { start } = reporter;
start();
```

then `start` would lose the reference to `this` being the `reporter` instance. This is to avoid such cases. There was an `if (this)` statement in the reporter that triggered me to change all the methods to lambdas for consistency and to avoid this happening in the future.

Reply:
sigh... I really dislike how JS does classes.... 

In other places in Kibana we use `bind` in the constructors to get around this, so my only poin was about uniformity, but this really isn't a blocker, especially considering the 7.5 push.